### PR TITLE
Major overhaul of landing page

### PR DESCRIPTION
**Pull request by alaframboise**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,44 @@
...
+
+![Process](./arcgis-process.png)
 
 ## Getting Started
 
-Sign up for a free [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/) to complete the labs. The subscription allows you to download software, register apps, import data, create new services, store maps and other content, perform spatial analysis and much, much more.
+Here's how to get started:
 
-> Note: ArcGIS Organizational accounts can also be used but a [free public account](https://geonet.esri.com/groups/geodev/blog/2014/10/15/help-i-cant-get-into-developersarcgiscom) cannot be used.
+1. **Sign up for a free [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/)** - With the subscription you candownload software, register apps, import data, create new services, store maps and other content, perform spatial analysis and more. If you have an ArcGIS Organizational account, you can use that as well, but please note that [Free public accounts](https://geonet.esri.com/groups/geodev/blog/2014/10/15/help-i-cant-get-into-developersarcgiscom) cannot be used.
```
Comment (nixta) [inline README.md]: `candownload` should be 2 words

File: README.md [hunk 2]
```diff
@@ -1,26 +1,44 @@
...
+
+2. **Start the labs** - The labs are divided into three sections that you can complete at your own pace. You can follow them in order or just choose the ones you want to complete. You can use the data we provide or use your own. If you want to code, dig into APIs and SDKs from Esri and compatible open source libraries. If not, you can still build highly customized production-ready apps using Esri's templates and builders. You might also find it helpful to follow along with the [Hackerlab Slides](http://slides.com/alaframboise/geodev-hackerlabs).
```
Comment (nixta) [inline README.md]: Should these slides be extracted and uploaded to a repo on github gh-pages (maybe even this repo) for view independent of your slides.com account?

Comment (alaframboise) [inline README.md]: Ya, for now I'm changing them quite a bit every day, but eventually they should. The issue is that they would be in reveal.js and not as easy to update/edit.

File: design/style_a_map.md [hunk 3]
```diff
@@ -8,39 +8,39 @@ In this lab you will use the styling capabilities of the viewer to make your map
...
 
-2. Click on [this link](http://www.arcgis.com/home/webmap/viewer.html?webmap=df0da19116934be89ac5f7379172c24e) to load the unstyled map.
+2. Click on [this link](http://www.arcgis.com/home/webmap/viewer.html?webmap=df0da19116934be89ac5f7379172c24e) to load the unstyled map or use your own unstyled map.
 
 3. Click `Save As` to save your own version of the map. Optionally `Share` > `Everyone`.
 
 4. Click on `Rail Stops` > `Change Style` and set the following:
 	
-	1. Choose an attribute to show: Type
+	1. 1 - Choose an attribute to show: `Type`
```
Comment (nixta) [inline design/style_a_map.md]: What's with this markup? It looks weird. The items are numbered twice (also applies to 4.2, 5.1, 5.2, 6.1 and 6.2).

File: README.md (hunk 2 above)
Comment (nixta) [inline README.md]: Cool. As long as it's on the roadmap :)

Comment (nixta) [inline README.md]: Oh, and I would think it's OK to edit them in slides.com but just host them here.

File: design/style_a_map.md (hunk 3 above)
Comment (alaframboise) [inline design/style_a_map.md]: I think we decided to only use ```xxx``` for clickable items. In this case the title label is exactly this: 1. Choose an attribute to show

**Pull request merged**